### PR TITLE
a few config changes

### PR DESCRIPTION
**Pull request by goncer**
* Add more tokens in mangata config.
* Update endpoint
* Log level to get node logs
* Create bifrost.yml
* Create imbu.yml

### CODE REVIEW

**Review by xlc — APPROVED** [review]

File: configs/bifrost.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+block: ${env.BIFROST_BLOCK_NUMBER}
+db: ./db.sqlite
+
+import-storage:
+  System:
+    Account:
+      -
+        -
+          - 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+        - data:
```
Comment (ermalkaleci) [inline configs/bifrost.yml:12]: @goncer please set providers to 1 so you don't have to updated this later

Comment (goncer) [reply configs/bifrost.yml:12]: what do you mean by providers? not sure I understand you :upside_down_face:

Comment (goncer) [reply configs/bifrost.yml]: updated :crossed_fingers:

Comment (ermalkaleci) [reply configs/bifrost.yml]: yes, latest substrate version requires providers so when you upgrade it will still work

**Pull request merged**